### PR TITLE
docs(onboarding): add local service dependency explainer

### LINKED ISSUE #1771: feat(onboarding): add local service dependency explainer
## Description
Implement an LLM-friendly, independently shippable backlog item for **add local service dependency explainer**. This issue should stay narrow enough for one agent/PR while producing durable behavior, tests, and documentation where appropriate.

## Affected locations
- docs/onboarding and contributor guides
- CLI/help output or onboarding scripts
- PM/worker handoff templates

## Impact
Without this work, future agents and operators will keep rediscovering the same gap, creating avoidable coordination cost, reliability risk, or ambiguous handoffs.

## Recommendation
Implement the scoped behavior with deterministic tests, concise docs, and structured output that PM/liveness tooling can consume.

## Goals
Deliver add local service dependency explainer as a discrete capability or hardening change.
Make the behavior deterministic enough for autonomous workers to verify.
Keep the implementation small enough for a one-issue worker and one PR.
Capture operator/user-facing behavior in docs or help text when applicable.

## Expected outcomes
add local service dependency explainer is available, documented, and test-backed.
Future PM/worker handoffs can point at this issue without needing extra interpretation.
Failure modes are explicit and produce actionable messages instead of silent drift.
The change does not broaden into adjacent backlog work or unrelated refactors.

## Suggested implementation slices
Inventory the current code/docs paths that own this behavior.
Add or update the narrow model/config/schema needed for the feature or guardrail.
Implement the smallest runtime/docs change that satisfies the acceptance criteria.
Add focused fixtures/tests for success, failure, and edge cases.
Update docs/help output and handoff notes so later agents can use it.

## Acceptance criteria
- [ ] There is a dedicated implementation or documentation change for add local service dependency explainer.
- [ ] At least one automated test, fixture, or deterministic verifier covers the intended behavior.
- [ ] At least one negative/edge case is covered so the failure mode is explicit.
- [ ] Docs/help text or inline operator guidance explains how to use or interpret the behavior.
- [ ] The PR description links this issue and summarizes verification evidence.
- [ ] No unrelated production behavior is changed outside the stated scope.

## LLM handoff notes
- Treat this as a one-issue/one-PR unit.
- Start by searching for existing implementations and tests before adding new surfaces.
- Prefer structured outputs and fixtures over prose-only behavior.
- If this overlaps an existing open issue, comment with evidence and stop for PM deduplication instead of creating an unscoped implementation.
- Do not perform broad refactors or change unrelated worker/PM behavior while implementing this ticket.

## Source
Created by the 2026-07-11 fourth strategic backlog expansion run requesting 100 issues. Topic: **Onboarding**.

### PR DESCRIPTION
**Pull request by djm204**
## Summary
- Add a local service dependency explainer for ChromaDB, Grafana, Tempo, provider credentials, and secret backends
- Add a structured manifest for stable service ids, health checks, edge cases, and PM/worker handoffs
- Link the explainer from onboarding and quickstart docs

## Test Plan
- npm run test:root -- --run tests/docs-issue-1771.test.ts
- npm run lint
- npm run typecheck
- npm run build

Closes #1771

### CODE REVIEW

**Comment by djm204** [comment]
@codex review

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+  "schemaVersion": 1,
+  "defaultPolicy": "Most local services are optional. Start only the service tied to the capability you are testing, then verify its health before blaming provider or agent code.",
+  "services": [
+    {
+      "id": "chromadb",
+      "name": "ChromaDB",
+      "requiredFor": ["semantic memory", "local memory seeding", "memory-backed MCP workflows"],
+      "notRequiredFor": ["workspace install", "root unit tests", "orchestrator CLI help", "dashboard shell startup"],
+      "startCommand": "docker compose up -d chroma",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Fixed in 310b2505: the manifest and prose now use the actual compose service name `chromadb`, matching docker-compose.yml and scripts/seed.ts.

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      "healthCheck": "curl -fsS http://localhost:8000/api/v2/heartbeat || curl -fsS http://localhost:8000/api/v1/heartbeat",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Fixed in 310b2505: the Chroma health check now requires `/api/v2/heartbeat` only, so a v1-only endpoint no longer passes this structured dependency check.

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      "failureSymptom": "Provider resolution fails, local chat cannot produce a turn, or Beast runs stop before model invocation.",
+      "handoffNote": "Name the selected provider path and whether it is CLI-authenticated or API-key backed."
+    },
+    {
+      "id": "secret-backend",
+      "name": "Secret backend",
+      "requiredFor": ["operator token lookup", "stored provider credentials", "Beast controls after init"],
+      "notRequiredFor": ["repository bootstrap", "docs-only checks", "unit tests with injected secrets"],
+      "startCommand": "node packages/franken-orchestrator/dist/cli/run.js init",
+      "healthCheck": "test -f .fbeast/config.json && (test -f .fbeast/secrets.enc || true)",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Fixed in 310b2505: removed the always-true fallback and replaced it with backend-specific proof for local encrypted, Bitwarden, 1Password, or OS keychain availability.

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 4]
```diff
@@ -0,0 +1,75 @@
+{
+  "schemaVersion": 1,
+  "defaultPolicy": "Most local services are optional. Start only the service tied to the capability you are testing, then verify its health before blaming provider or agent code.",
+  "services": [
+    {
+      "id": "chromadb",
+      "name": "ChromaDB",
+      "requiredFor": ["semantic memory", "local memory seeding", "memory-backed MCP workflows"],
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Fixed in 310b2505: the Chroma requirement is now limited to semantic-memory seed/verification scripts, and file-backed MCP memory is explicitly listed as not requiring ChromaDB.

**Comment by djm204** [comment]
@codex review

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+      "failureSymptom": "Provider resolution fails, local chat cannot produce a turn, or Beast runs stop before model invocation.",
+      "handoffNote": "Name the selected provider path and whether it is CLI-authenticated or API-key backed."
+    },
+    {
+      "id": "secret-backend",
+      "name": "Secret backend",
+      "requiredFor": ["operator token lookup", "stored provider credentials", "Beast controls after init"],
+      "notRequiredFor": ["repository bootstrap", "docs-only checks", "unit tests with injected secrets"],
+      "startCommand": "node packages/franken-orchestrator/dist/cli/run.js init",
+      "healthCheck": "test -f .fbeast/config.json && (test -f .fbeast/secrets.enc || test -n \"${BW_SESSION:-}\" || op account get >/dev/null 2>&1 || security find-generic-password -s frankenbeast >/dev/null 2>&1)",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Fixed in 08297b55: the secret backend check now reads .fbeast/config.json, branches on network.secureBackend, requires FRANKENBEAST_PASSPHRASE for local-encrypted, BW_SESSION for bitwarden, an op account probe for 1password, and no longer succeeds on an unrelated stale secrets file.

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+      "env": ["CHROMA_URL=http://localhost:8000"],
+      "failureSymptom": "Semantic memory seed or verification scripts fail with connection refused, unavailable Chroma endpoint, or missing v2 tenant/database API errors.",
+      "handoffNote": "Mention whether CHROMA_URL is set, whether the v2 heartbeat responded, and whether the work uses semantic-memory scripts rather than file-backed MCP memory."
+    },
+    {
+      "id": "grafana",
+      "name": "Grafana",
+      "requiredFor": ["local dashboards", "operator-facing observability UI"],
+      "notRequiredFor": ["trace collection", "root unit tests", "CLI planning and run commands"],
+      "startCommand": "GRAFANA_USER=admin GRAFANA_PASSWORD=<unique-password> docker compose up -d grafana",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Fixed in 08297b55: the manifest and guide now use the shell-safe placeholder value replace-with-unique-password instead of <unique-password>, and the docs test asserts the unsafe shell-redirection placeholder is absent.

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+      "failureSymptom": "Trace export or Grafana trace panels cannot reach Tempo, while local SQLite traces may still be present.",
+      "handoffNote": "Check both Tempo readiness on 3200 and OTLP/HTTP target 4318 when debugging trace export."
+    },
+    {
+      "id": "provider-cli",
+      "name": "AI provider CLI or API credentials",
+      "requiredFor": ["local chat", "agent execution", "dashboard chat turns", "Beast runs against real providers"],
+      "notRequiredFor": ["documentation tests", "static typecheck", "most package unit tests"],
+      "startCommand": "Install and authenticate at least one configured provider CLI, or export the API key for an API-backed provider.",
+      "healthCheck": "command -v claude || command -v codex || command -v gemini || test -n \"${ANTHROPIC_API_KEY:-}${OPENAI_API_KEY:-}${GOOGLE_API_KEY:-}${GEMINI_API_KEY:-}\"",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Fixed in 08297b55: the provider health check no longer treats command -v as sufficient; it now requires a real smoke call through the selected configured provider and an authenticated no-op prompt for CLI-backed providers.

File: docs/onboarding/local-service-dependencies.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+
+- Start only when you are using semantic memory locally:
+
+  ```bash
+  docker compose up -d chromadb
+  export CHROMA_URL=http://localhost:8000
+  curl -fsS http://localhost:8000/api/v2/heartbeat
+  ```
+
+- Required for: `npm run local:seed`, `npm run local:verify-setup`, and semantic-memory verification that talks to ChromaDB.
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.md]: Fixed in 08297b55: the guide now treats npm run local:verify-setup as a full optional-stack probe for ChromaDB, Grafana, and Tempo, and semantic-memory-only users are pointed at the Chroma heartbeat/seed path instead.

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+  "schemaVersion": 1,
+  "defaultPolicy": "Most local services are optional. Start only the service tied to the capability you are testing, then verify its health before blaming provider or agent code.",
+  "services": [
+    {
+      "id": "chromadb",
+      "name": "ChromaDB",
+      "requiredFor": ["semantic memory scripts", "local memory seeding", "semantic-memory verification"],
+      "notRequiredFor": ["workspace install", "root unit tests", "orchestrator CLI help", "dashboard shell startup", "file-backed MCP memory"],
+      "startCommand": "docker compose up -d chromadb",
+      "healthCheck": "curl -fsS http://localhost:8000/api/v2/heartbeat",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Fixed in 08297b55: the Chroma health check now sources .env when present and probes ${CHROMA_URL:-http://localhost:8000}/api/v2/heartbeat, so externally managed or non-default Chroma endpoints are respected.

**Comment by djm204** [comment]
@codex review

**Comment by djm204** [comment]
@codex review

**Comment by djm204** [comment]
@codex review

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+      "handoffNote": "Name the selected provider path and whether it is CLI-authenticated or API-key backed."
+    },
+    {
+      "id": "secret-backend",
+      "name": "Secret backend",
+      "requiredFor": ["operator token lookup", "stored provider credentials", "Beast controls after init"],
+      "notRequiredFor": ["repository bootstrap", "docs-only checks", "unit tests with injected secrets"],
+      "startCommand": "node packages/franken-orchestrator/dist/cli/run.js init",
+      "healthCheck": "node -e \"const fs=require('node:fs'),cp=require('node:child_process'); const cfg=JSON.parse(fs.readFileSync('.fbeast/config.json','utf8')); const b=cfg.network?.secureBackend ?? 'local-encrypted'; if (b === 'local-encrypted') { if (!fs.existsSync('.fbeast/secrets.enc') || !process.env.FRANKENBEAST_PASSPHRASE) process.exit(1); } else if (b === 'bitwarden') { if (!process.env.BW_SESSION) process.exit(1); } else if (b === '1password') { cp.execFileSync('op',['account','get'],{stdio:'ignore'}); } else if (b === 'os-keychain') { process.exit(0); } else process.exit(1);\"",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Updated the manifest health check to instantiate LocalEncryptedStore and call keys(), so a wrong FRANKENBEAST_PASSPHRASE now fails during decryption instead of passing on file/env presence only. Fixed in e5802b8b.

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+      "env": ["CHROMA_URL=http://localhost:8000"],
+      "failureSymptom": "Semantic memory seed or verification scripts fail with connection refused, unavailable Chroma endpoint, or missing v2 tenant/database API errors.",
+      "handoffNote": "Mention whether CHROMA_URL is set, whether the v2 heartbeat responded, and whether the work uses semantic-memory scripts rather than file-backed MCP memory."
+    },
+    {
+      "id": "grafana",
+      "name": "Grafana",
+      "requiredFor": ["local dashboards", "operator-facing observability UI"],
+      "notRequiredFor": ["trace collection", "root unit tests", "CLI planning and run commands"],
+      "startCommand": "GRAFANA_USER=admin GRAFANA_PASSWORD=replace-with-unique-password docker compose up -d grafana",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Updated the Grafana start command to use docker compose up -d --no-deps grafana so following the manifest does not start Tempo implicitly. Fixed in e5802b8b.

File: docs/onboarding/local-service-dependencies.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+  curl -fsS http://localhost:3200/ready
+  ```
+
+- Required for: OTLP trace export smoke tests and Grafana trace exploration.
+- Not required for: SQLite observer adapters, most package tests, or MCP initialization.
+- Edge case: verify readiness on port `3200` and the OTLP/HTTP target on `4318` when debugging trace export.
+
+### Provider CLI or API-backed providers
+
+- Local chat, Beast runs, and dashboard chat need at least one real provider path. Install and authenticate a supported CLI (`claude`, `codex`, or `gemini`) or configure an API-backed provider with an exported key such as `ANTHROPIC_API_KEY`, `OPENAI_API_KEY`, `GOOGLE_API_KEY`, or `GEMINI_API_KEY`.
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.md]: Scoped API-key guidance to provider-registry Beast runs and documented that chat/dashboard chat paths require CLI-backed providers through the CLI provider registry. Fixed in e5802b8b.

File: docs/onboarding/local-service-dependencies.manifest.json (hunk 11 above)
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Changed the Grafana command to generate a random password inline with openssl rand instead of using a copyable placeholder password. Fixed in e5802b8b.

**Comment by djm204** [comment]
@codex review

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+      "requiredFor": ["operator token lookup", "stored provider credentials", "Beast controls after init"],
+      "notRequiredFor": ["repository bootstrap", "docs-only checks", "unit tests with injected secrets"],
+      "startCommand": "node packages/franken-orchestrator/dist/cli/run.js init",
+      "healthCheck": "node --input-type=module -e \"import fs from 'node:fs'; import cp from 'node:child_process'; const cfg=JSON.parse(fs.readFileSync('.fbeast/config.json','utf8')); const b=cfg.network?.secureBackend ?? 'local-encrypted'; if (b === 'local-encrypted') { if (!fs.existsSync('.fbeast/secrets.enc') || !process.env.FRANKENBEAST_PASSPHRASE) process.exit(1); const { LocalEncryptedStore } = await import('./packages/franken-orchestrator/dist/network/secret-backends/local-encrypted-store.js'); await new LocalEncryptedStore({ projectRoot: process.cwd(), passphrase: process.env.FRANKENBEAST_PASSPHRASE }).keys(); } else if (b === 'bitwarden') { if (!process.env.BW_SESSION) process.exit(1); } else if (b === '1password') { cp.execFileSync('op',['account','get'],{stdio:'ignore'}); } else if (b === 'os-keychain') { process.exit(0); } else process.exit(1);\"",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Fixed in cbd52229. The manifest now exercises the configured backend instead of accepting env-only checks: local-encrypted still decrypts via LocalEncryptedStore.keys(), Bitwarden requires BW_SESSION plus BitwardenStore detection and a real `bw list items --search frankenbeast` probe so stale sessions fail, and os-keychain now instantiates OsKeychainStore and runs platform detection/keys so missing secret-tool/security/cmdkey fails the health check. Added targeted assertions in tests/docs-issue-1771.test.ts.

File: docs/onboarding/local-service-dependencies.md [hunk 14]
```diff
@@ -0,0 +1,103 @@
...
+  ```
+
+- Required for: OTLP trace export smoke tests and Grafana trace exploration.
+- Not required for: SQLite observer adapters, most package tests, or MCP initialization.
+- Edge case: verify readiness on port `3200` and the OTLP/HTTP target on `4318` when debugging trace export.
+
+### Provider CLI or API-backed providers
+
+- Local chat, agent execution, and dashboard chat surfaces currently resolve providers through the CLI provider registry. Install and authenticate a supported CLI (`claude`, `codex`, or `gemini`) before expecting those chat paths to start.
+- Provider-registry Beast runs may use either a CLI-backed provider or an API-backed provider with an exported key such as `ANTHROPIC_API_KEY`, `OPENAI_API_KEY`, `GOOGLE_API_KEY`, or `GEMINI_API_KEY`; scope API-key-only guidance to those Beast/provider-registry paths.
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.md]: Fixed in cbd52229. The provider guidance now says normal `frankenbeast run`/agent execution, local chat, and dashboard chat require a CLI-backed provider login; API-key-only setup is scoped only to explicit API-backed provider-registry integrations and the fast table explicitly says not to treat normal `frankenbeast run` as API-key-only. Added targeted regression assertions in tests/docs-issue-1771.test.ts.

**Comment by djm204** [comment]
@codex review

File: docs/onboarding/local-service-dependencies.manifest.json (hunk 13 above)
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json]: Fixed in 897e5285: the manifest health check now constructs the runtime secret store via createSecretStore(), requires detect()/keys(), resolves every configured secret ref with store.resolve(ref), and runs SecretResolver.resolveAll(config) so Bitwarden/OS-keychain/local-encrypted checks exercise the same backend path used at runtime instead of reporting healthy from presence checks alone.

File: docs/onboarding/local-service-dependencies.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.md]: Fixed in 897e5285: the provider section now documents CLI credentials as required for normal Beast run/chat paths, explicitly warns that anthropic-api/openai-api/gemini-api still fail CLI-registry preflight on those surfaces, and limits API-key-only guidance to low-level adapter integrations that bypass normal Beast run/chat.

**Comment by djm204** [comment]
@codex review

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+      "healthCheck": "node --input-type=module -e \"import fs from 'node:fs'; const cfg=JSON.parse(fs.readFileSync('.fbeast/config.json','utf8')); const backend=cfg.network?.secureBackend ?? 'local-encrypted'; const { createSecretStore, SecretResolver } = await import('./packages/franken-orchestrator/dist/network/secret-store.js'); const store=createSecretStore(backend,{projectRoot:process.cwd(),passphrase:process.env.FRANKENBEAST_PASSPHRASE}); const detection=await store.detect(); if(!detection.available) throw new Error(detection.reason ?? ('secret backend '+backend+' unavailable')); await store.keys(); const refs=[cfg.network?.operatorTokenRef,cfg.comms?.orchestratorTokenRef,cfg.comms?.slack?.botTokenRef,cfg.comms?.slack?.signingSecretRef,cfg.comms?.discord?.botTokenRef,cfg.comms?.telegram?.botTokenRef,cfg.comms?.whatsapp?.accessTokenRef,cfg.comms?.whatsapp?.appSecretRef,cfg.comms?.whatsapp?.verifyTokenRef].filter(Boolean); for (const ref of refs) { const value=await store.resolve(ref); if(!value) throw new Error('secret ref '+ref+' did not resolve through '+backend); } await new SecretResolver(store).resolveAll(cfg);\"",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json:55]: Fixed in follow-up PR #2564, which has been merged after green CI and a fresh current-head Codex clean review. The follow-up updates the local-service dependency manifest/docs/tests for this finding.

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+      "env": ["OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318"],
+      "failureSymptom": "Trace export or Grafana trace panels cannot reach Tempo, while local SQLite traces may still be present.",
+      "handoffNote": "Check both Tempo readiness on 3200 and OTLP/HTTP target 4318 when debugging trace export."
+    },
+    {
+      "id": "provider-cli",
+      "name": "AI provider CLI credentials",
+      "requiredFor": ["local chat through CLI-backed providers", "agent execution", "dashboard chat turns through CLI-backed providers", "normal frankenbeast run paths"],
+      "notRequiredFor": ["documentation tests", "static typecheck", "most package unit tests"],
+      "startCommand": "Install and authenticate a configured CLI provider (claude, codex, or gemini) for chat surfaces and normal frankenbeast run/agent execution. API keys are only sufficient for code paths that explicitly bypass the CLI registry and construct API-backed provider adapters directly.",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json:43]: Fixed in follow-up PR #2564, which has been merged after green CI and a fresh current-head Codex clean review. The follow-up updates the local-service dependency manifest/docs/tests for this finding.

File: docs/onboarding/local-service-dependencies.manifest.json [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+      "failureSymptom": "Dashboard links or Grafana panels are unavailable even though application tests and traces still run.",
+      "handoffNote": "Record that Grafana credentials were set to a non-default password before starting compose."
+    },
+    {
+      "id": "tempo",
+      "name": "Tempo",
+      "requiredFor": ["local distributed trace viewing", "OTLP trace export smoke tests"],
+      "notRequiredFor": ["SQLite observer storage", "root unit tests", "MCP initialization"],
+      "startCommand": "docker compose up -d tempo",
+      "healthCheck": "curl -fsS http://localhost:3200/ready",
```
[the remark this answers is not in the document]
Comment (djm204) [reply docs/onboarding/local-service-dependencies.manifest.json:33]: Fixed in follow-up PR #2564, which has been merged after green CI and a fresh current-head Codex clean review. The follow-up updates the local-service dependency manifest/docs/tests for this finding.
